### DISCUSSION TITLE
Route Rules proxy

### DISCUSSION THREAD

**Discussion by SulivanNext** [nuxt/nuxt#27451] [Questions]
hello guys! I would like to know if it is a bad practice or opens up some vulnerability to use the routeRules proxy to bypass CORS in my backend and set the session correctly, which has been an extreme sacrifice to make it work in SSR. It works perfectly, but are there any cons to it? Does it open up vulnerabilities?
 thanks in advance.

**Comment by chibx** [comment]
@SulivanNext From experience, after trying to set allowed domains for my google console account, I mistakenly set my cors to allow requests from any localhost, and even set it to true. Well, I can't say much, but unless you really trust (like really trust) a domain, you should always use cors to prevent unwanted sites
